src/components/MetricsPane/SelectedSiteMetrics.styles.js

```diff
@@ -98,3 +104,28 @@ export const SelectedSiteActionBar = styled(Row)`
     width: 100%;
   }
 `
+
+export const TabButtonContainer = styled.div`
+  display: flex;
+  width: 100%;
+`
+export const TabButton = styled.button`
+  all: unset;
+  padding: 0.5rem;
+  background-color: ${(props) =>
+    props.$isSelected ? theme.color.primaryColor : theme.color.grey0};
+  color: ${(props) => (props.$isSelected ? theme.color.white : theme.color.black)};
+  width: 100%;
+  text-align: center;
+`
+export const TabContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  height: 100%;
+  overflow-y: auto;
```

Comment:
@alanjleonard take note of some exec decisions I made with the metrics metadata scrollbar.

Reply:
<img width="601" alt="Screenshot 2024-09-25 at 1 47 52 PM" src="https://github.com/user-attachments/assets/7d5a2ff9-914a-44b6-9ea8-21978d0eceb7">

Reply:
Why not scroll the whole pane like the filter pane? This could be an issue on shorter browsers since the scrollable space can be quite small. (We've seen a lot of people with short browser windows in user testing)

Reply:
Also the graphs may not fit on mobile if we fix the survey info at the top. Here's a mockup of a graph on my S10. You can see that the x axis label is cut off. 
![image](https://github.com/user-attachments/assets/977ee663-50df-4d8b-b5a1-d309ce13b4db)

Reply:
Sure, scrolling the whole pane works. Im going to circle back to this though with its own ticket or it might end up in a backlog ticket depending on my project assignments next week.

Reply:
https://trello.com/c/wAUa0g0U/1061-metrics-pane-style-tweaks?filter=label:dash%20app